### DISCUSSION TITLE
Create schedule in HA?

### DISCUSSION THREAD

**Discussion by martinalbinger** [mtrossbach/noah-mqtt#30] [Ideas]
Hi Markus,
great implementation! I love it. worked flawlessly on the first try.
I just have a new idea:
Would it be possible to create the schedule in HA instead of the complicated way in the Shine app?
I use the Noah for covering the night consumption and it would be helpful to have an easy way of adjusting the output power more flexible than in the shine app, maybe even with weekly and/or monthly schedules?

Thanks an let me know what you think and continue with the great work
Martin

**Comment by RealGue** [comment]
Hi Martin,
it should already be possible within HA. You could use time-triggered Automations and set the System Output Power of the Noah device. Or try Helper/Schedule. 
Jörn

**Reply by martinalbinger** [reply]
Thanks a lot.

this did the trick

**Reply by les-les** [reply]
I m chasing the same goal 
Can you helpmme out wirh some information how you did it in homeassistant

**Reply by martinalbinger** [reply]
Hi,
 I simply followed the sugestion to create several automations by time in the mqtt settings for the noah. It worked as expected, but was not as comfortable to set up as I wanted. Currently i control the noah simply by time from 18:00 to 6:00 using 100 Watt as output.
